doc/hand_eye_calibration/hand_eye_calibration_tutorial.rst

```diff
@@ -0,0 +1,142 @@
+Hand-Eye Calibration
+====================
+The `MoveIt Calibration <http://www.github.com/ros-planning/moveit_calibration>`_ package provides plugins and a graphical
+interface for conducting a hand-eye camera calibration. Calibrations can be performed for cameras rigidly mounted in the
+robot base frame (eye-to-hand) and for cameras mounted to the end effector (eye-in-hand). This tutorial presents the
+eye-in-hand case.
+
+.. image:: images/hand_eye_calibration_demo.jpg
+
+Getting Started
+---------------
+While it is possible to go through most of this tutorial using just a simulation, to actually complete a calibration you
+will need a robotic arm and a camera.
+
+If you haven't already done so, be sure to complete the steps in `Getting Started
+<../getting_started/getting_started.html>`_. Also, set your arm up to work with MoveIt (as described in the `Setup
+Assistant Tutorial <../setup_assistant/setup_assistant_tutorial.html>`_).
+
+This tutorial also requires a camera, publishing images and a ``sensor_msgs/CameraInfo`` topic with good intrinsic
+calibration parameters and an accurate coordinate frame. (Conduct an intrinsic camera calibration by using the
+`camera_calibration <http://wiki.ros.org/camera_calibration>`_ package, if necessary.)
+
+Clone and Build the MoveIt Calibration Repo
+-------------------------------------------
+In your workspace ``src`` directory, clone MoveIt Calibration::
+
+  git clone git@github.com:ros-planning/moveit_calibration.git
+
+Then, make sure you have the appropriate dependencies and build the package::
+
+  rosdep install -y --from-paths . --ignore-src --rosdistro melodic
+  catkin build
+  source devel/setup.sh
+
+Launch RViz and Load Calibration Plugin
+---------------------------------------
+Launch the appropriate MoveIt demo for your robot. For instance, ``roslaunch panda_moveit_config demo.launch``.
```

Comment:
This robot doesn't actually have a camera, right? Is there a reasonably "standard" moveit config for an arm that already contains a camera?

Reply:
That is correct--there's no camera on the Panda. I don't know if there is a "standard" config with a camera, but we'd need Gazebo to render the camera images anyway. I have put together a calibration simulation, which I think needs some polishing before sharing, but I'm thinking a future version of this tutorial could use it. That way, people can go through the tutorial without hardware.

Reply:
Would be very helpful. When you're learning and figuring things out, too many layers of "Ah yeah this part is just pretend, you can ignore it" can be very confusing.

To be honest, even setting up the robot and the camera is not easy and would be worth explaining. Linking to summaries of the relevant steps would be a good alternative (URDF, camera driver, camera topics...).